.github/workflows/quality-gates.yml

```
@@ -0,0 +1,70 @@
+        uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm ci
+      - name: Build
+        run: npm run build
+      - name: Calculate dist size
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2046:warning:2:21: Quote this to prevent word splitting

(shellcheck)